<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color:rgb(41, 175, 97)}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color:green}</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(230, 120, 220)}</string>
    </property>
    <property name="text">
     <string>Local IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="localIP">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="localPort">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(230, 120, 220)}</string>
    </property>
    <property name="text">
     <string>Local Port#</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remoteIP">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(230, 120, 220)}</string>
    </property>
    <property name="text">
     <string>Remote IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remotePort">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="text">
     <string>3000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color:rgb(230, 120, 220)}</string>
    </property>
    <property name="text">
     <string>Remote Port#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color:rgb(49, 35, 255);}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>361</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="display">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{background-color:rgb(255, 199, 200)}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color:red}</string>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
